explorer/db/ent/schema/receipt.go

```diff
@@ -1,6 +1,11 @@
 package schema
 
 import (
+	"context"
+	gen "github.com/omni-network/omni/explorer/db/ent"
+	"github.com/omni-network/omni/explorer/db/ent/hook"
+	"github.com/omni-network/omni/explorer/db/ent/msg"
+	"github.com/omni-network/omni/lib/errors"
 	"time"
```

Comment:
nit; shouldn't the imports be ordered automatically?

Reply:
weird, let me check my editor settings